### PR TITLE
Summer intern

### PR DESCRIPTION
**Pull request by trekrabbor**
There is room for improvement on this code. Right now, it doesn't show an ugly error message if there is something wrong with the yaml formatting.

### CODE REVIEW

File: src/catalog.js [hunk 1]
```diff
@@ -237,7 +239,14 @@ class Page {
   }
 
   static create({ category, parent, slug, sources }) {
-    const matter = frontMatter(sources);
+    var matter;
```
Comment (spacedsweden) [inline src/catalog.js:242]: here you set matter to sources, on row 242 you reset it, change it to one variable.

File: src/catalog.js [hunk 2]
```diff
@@ -237,7 +239,14 @@ class Page {
...
+
+    try {
+      matter = frontMatter(sources);
+    } catch(error){
+      console.error(error.name + ": Check the YAML formatting of the file: " + chalk.cyan(category + "/" + slug + ":" + error.mark.line) + ", by using a linter.");
```
Comment (spacedsweden) [inline src/catalog.js:247]: I think to you need to add parent as well if its not null

Comment (trekrabbor) [reply src/catalog.js:247]: The parent is undefined when you try to. console log it.

File: src/catalog.js (hunk 1 above)
Comment (trekrabbor) [reply src/catalog.js:242]: Jag tog bort const matter = frontMatter(sources) och ersatte den med var matter. Det enda som är med är var matter.

File: src/catalog.js (hunk 2 above)
Comment (spacedsweden) [reply src/catalog.js:247]: not alwasy, you need to check if its undefined. its only not defined when the page is in a subfolder.

File: src/catalog.js (hunk 1 above)
Comment (spacedsweden) [reply src/catalog.js:242]: lets make a new pull request then

**Pull request closed without merging**